<!--
  Author: David Kent, Worcester Polytechnic Institute
  Version: March 18, 2013
-->

<launch>
  <!-- start the grasp collection nodes -->
  <node pkg="rail_grasp_collection" type="object_tracker" name="object_tracker" />
  <node pkg="rail_grasp_collection" type="rail_grasp_collection" name="rail_grasp_collection" />
</launch>

